### ISSUE TITLE
Description entered in the "Enter the description" phase is not added to the update-descriptor3.yaml file

### ISSUE THREAD

**Issue by this** [wso2/update-creator-tool#71]
**Description:**
$subject. Here is the console output.
```
Enter the description: This update fixes an UI element issue occur in Firefox browser in the dashboard listing page of the Portal webapp

'update-descriptor.yaml' has been successfully created in '/home/sajith/patch-tools/wum-uc/creating-update'.
Optional resource file 'update-descriptor3.yaml' not copied.
Optional resource file 'instructions.txt' not copied.
'update-descriptor3.yaml' has been successfully created in '/home/sajith/patch-tools/wum-uc/creating-update'.
'WSO2-CARBON-UPDATE-5.0.0-0013.zip' successfully created.

Your update applies to the following products
	Compatible products : [wso2sp-4.2.0.full] 
	Partially applicable products : [] 
	Notify products : [] 
Manually fill the `description`,`instructions` and `bug_fixes` fields for above products in the update-descriptor3.yaml located inside the created 'WSO2-CARBON-UPDATE-5.0.0-0013.zip'
```

**Suggested Assignees:**
@kasunsiyambalapitiya 

**Affected Product Version:**
3.0.1

**OS, DB, other environment details and versions:**    
wum-uc version: 3.0.2
Release date: 2018-08-09 09:48:34
OS\Arch: linux\amd64
Go version: go1.8

**Comment by kasunsiyambalapitiya** [comment]
There are two reasons for the above,

1) The tool will generate both the `update-descriptor.yaml` and `update-descriptor3.yaml` till `WUM 2.0` gets officially depreciated ( which will roughly be within next month) as the updates created from `wum-uc` should be full compatible with both `WUM 2.0` and `WUM 3.0.0` clients. So untill `WUM 2.0` gets officially depreciated the `wum-uc` will request you for inputs to `applies_to`, `bug_fixes` and `description` fields of the `update-descriptor.yaml` (used by `WUM 2.0` client) when creating an update. Once `WUM 2.0` gets officially depreciated requests for user inputs for the above fields will not be asked and hence the developer needs to manually fill in `description`, `instructions` and `bug_fixes` fields of the `update-descriptor3.yaml`

2) The reason for not filling relevant fields (`description` and `bug_fixes`) of `update-descriptor3.yaml` from above-obtained user inputs is because from `WUM 3.0.0` onwards we support partial updates. So even though your are creating an update for `WSO2AM-250` product, the same update may be applied fully to `WSO2IS-560` product and may also be partially applied to `WSO2MB-3.2.0` product. Hence in the `update-descriptor3.yaml` the `description`, `instructions` and `bug_fixes` fields should contain details for all the above products and thus the developer creating the above update should manually enter them. So in the process of update creation, the tool will fill in default values for the above fields and checks whether they have been altered by the developer in the update validation process.

so as per the above reasons details entered in "Enter the description", "Enter Bug fixes" phases are not added to the `update-descriptor3.yaml` file

**Comment by this** [comment]
OK, So after deprecating WUM v2.0, `description` and `bug_fixes` will be added to the `update-descriptor.yaml`, right?

**Comment by kasunsiyambalapitiya** [comment]
No, as already mentioned in the first point of my previous comment, after `WUM 2.0` gets depreciated user inputs for `description`, `bug_fixes` and `instructions` will not be asked in the interactive mode when creating an update and the developer needs to manually fill in values for the above fields in the `update-descriptor3.yaml` as we support partial updates from `wum 3.0.0` onwards.
